include/drivers/espi_emul.h

```diff
@@ -0,0 +1,138 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef ZEPHYR_INCLUDE_DRIVERS_ESPI_SPI_EMUL_H_
+#define ZEPHYR_INCLUDE_DRIVERS_ESPI_SPI_EMUL_H_
+
+/**
+ * @file
+ *
+ * @brief Public APIs for the eSPI emulation drivers.
+ */
+
+#include <zephyr/types.h>
+#include <device.h>
+
+/**
+ * @brief eSPI Emulation Interface
+ * @defgroup espi_emul_interface eSPI Emulation Interface
+ * @ingroup io_emulators
+ * @{
+ */
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+#define EMUL_ESPI_HOST_CHIPSEL 0
+
+struct espi_emul;
+
+/**
+ * Passes eSPI virtual wires set request to the emulator.
+ * The emulator updates the level of vw.
+ *
+ * @param emul Emulator instance
+ * @param vw The signal to be set.
+ * @param level The level of the signal to be set.
```

Comment:
What is a level? What are the valid values? Why is this u8?

Reply:
I added a comment with the valid values.
u8 is used to be consistent with the current eSPI API -
`int espi_send_vwire(const struct device *dev, enum espi_vwire_signal signal, uint8_t level);
`